evm/src/cpu/gas.rs

```diff
@@ -70,20 +70,22 @@ fn eval_packed_accumulate<P: PackedField>(
         })
         .sum();
 
-    let constr = nv.gas - (lv.gas + gas_used);
+    let gas_diff = nv.gas[1] * P::Scalar::from_canonical_u64(1 << 32) + nv.gas[0]
+        - (lv.gas[1] * P::Scalar::from_canonical_u64(1 << 32) + lv.gas[0]);
+    let constr = gas_diff - gas_used;
```

Comment:
Could overflowing the field size be an issue here?

Reply:
Hmm could be. The largest gas_used in the EVM test suite is 53 bits long, so we'd be fine even in this extreme case.
Couldn't we just add a specific TODO here saying that before prod, this should be reverted to only 1 limb?

Reply:
> Couldn't we just add a specific TODO here saying that before prod, this should be reverted to only 1 limb?

Yes sounds good 👍